src/constants/colors.ts

```diff
@@ -0,0 +1,2 @@
+
+export const COLORS: Array<string> = ['#37d67a', '#2ccce4', '#06A77D', '#ff8a65', '#1E91D6']
```

Comment:
We can turn `prefer default export` out.

Reply:
You know, I don't mind named imports. What do you think?